### PR TITLE
IA-3143 Update terra-jupyter-r image to Bioc 3.14, R 4.1.2

### PR DESCRIPTION
**Pull request by nturaga**
Update Terra Jupyter image to use latest Bioconductor 3.14.

https://broadworkbench.atlassian.net/browse/IA-3143

### CODE REVIEW

**Comment by nturaga** [comment]
Not sure what other files have to change. It seems the CHANGELOG files are autogenerated now. Is that correct? 

Can someone let me know what other files I'd have to change or what other images I'd have to rebuild? @Qi77Qi or @gpcarr

**Comment by xuscrbw** [comment]
Hi Nitesh,
To generate updates to the changelog, we use the following script to help us:
amm ./updateVersions.sc terra-jupyter-r <release note>

Once you do this, it should update the changelog for the image you changed and any images that depend on it.

If you are having trouble with running this, I can run this on my end and push the updated changelog to this branch (if I am able)

Sincerely,
Wilson

**Comment by xuscrbw** [comment]
You will also have to update config.json file as well (don't think the script will update this). I can help you with this part if needed

**Comment by nturaga** [comment]
> Hi Nitesh, To generate updates to the changelog, we use the following script to help us: amm ./updateVersions.sc terra-jupyter-r "release note"
> 
> Once you do this, it should update the changelog for the image you changed and any images that depend on it.
> 
> If you are having trouble with running this, I can run this on my end and push the updated changelog to this branch (if I am able)
> 
> Sincerely, Wilson

Please run the script if you can. I appreciate it @xuscrbw.

And also update the config file, because it's not clear to me.

**Review by Qi77Qi — APPROVED** [review]
thanks @xuscrbw !..assuming tests will pass

File: terra-jupyter-r/Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ COPY scripts $JUPYTER_HOME/scripts
 
 # Add env vars to identify binary package installation
 ENV TERRA_R_PLATFORM="terra-jupyter-r"
-ENV TERRA_R_PLATFORM_BINARY_VERSION=3.13.0
+ENV TERRA_R_PLATFORM_BINARY_VERSION=3.14.0
```
Comment (gpcarr) [inline terra-jupyter-r/Dockerfile:8]: we should also update the [readme](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-r/README.md#image-contents) for this dockerfile to reflect the new version of bioconductor

Comment (xuscrbw) [reply terra-jupyter-r/Dockerfile:8]: Good point. I need to also update this in the smoke tests I think.

Comment (nturaga) [reply terra-jupyter-r/Dockerfile:8]: Done.

**Comment by nturaga** [comment]
I think the `rhdf5` can be removed as well if the smoke test to install is timing out. 

We can make a binary available once the image is ready, so folks actually installing this package will not see such wait times. 

Local testing shows 21 seconds (binary installation) vs over 300 seconds (as per the smoke test).

**Pull request merged**